ci/test

```diff
@@ -1,9 +1,9 @@
-#!/bin/bash -e
-# shellcheck disable=SC1091
+#!/usr/bin/env bash
+set -e
```

Comment:
I agree, but because I'm going to wait until I refactor this code, but that could have unpredictable side effects, and this is an existing setting that's been here for sometime.  I just rewrote it from a shebang flag to be shellcheck compliant.